### ISSUE TITLE
How was NVDA purchased by the bot trained on data from DOW30?

### ISSUE THREAD

**Issue by Ashok93** [llSourcell/ChatGPT_Trading_Bot#4]
In your tutorial and notebook, you have trained on DOW30 stocks and in your final purchase you have used NVDA (Nvidia) stock. NVDA stock is not even part of DOW30. How can the model even learn about NVDA performance without even training on the its historical data? Can you please explain this?

**Comment by abhiprojectz** [comment]
@Ashok93  Obviously, The whole thing is black as crap, & now it doesn't even open.

**Comment by coininstant** [comment]
Yeah, I wonder too. Maybe that's why his bot only made 1% on NVDA it wasn't trained. Brah only making 1% on NVDA doesn't say much, even a monkey throwing darts blindfolded could beat that... lolz
